### DISCUSSION TITLE
When bouncy castle will release for NIST offcial finalized PQC algorithm (ML-DSA, ML-KEM, SLH-DSA)

### DISCUSSION THREAD

**Discussion by biau83** [bcgit/bc-java#1860] [General]
Hi, what is the roughly estimation date of bouncy castle will release for NIST offcial finalized PQC algorithm?

TQ

**Comment by dghgit** [comment]
End of October, at least that's the plan. There's current betas for all 3 on https://downloads.bouncycastle.org/betas
